index.html

```diff
@@ -196,6 +196,39 @@ <h3 class="text1">Language</h3>
       </div>
     </div>
  </section>
+ <footer class="footer">
+<p class="subject">
+    I'm always<br> interested in hearing<br> about new projects,<br> so if you'd like to<br> chat please get in
+</p>
+<form action="https://formspree.io/f/mwkzpjrj" method="post" class="form">
+<div class="full-name">
+    <input type="text" name="name" id="name" maxlength="30" required placeholder="Full name">
+</div>
+<hr class="hor">
+<div class="email">
+    <input type="email" name="email" id="email" required placeholder="Email address">
+</div>
+<hr class="hor">
+<div class="textarea">
+    <textarea name="message" id="message" cols="45" rows="6" required maxlength="500">Enter text here</textarea>
+</div>
+<div class="button-cont">
+    <button type="submit"  class="buttonl">Get in touch</button>
+    </div>
+</form>
+
+<hr class="my-hor">
+<div class="social-media2">
+    <ul>
+        <li class="git"><a href=""><img src="image/github2.png" alt="github-logo"></a></li>
+        <li><a href=""><img src="image/linkedin2.png" alt="linked-in-logo"></a></li>
+        <li><a href=""><img src="image/peace2.png" alt="peace2-logo"></a></li>
+        <li><a href=""><img src="image/twitter2.png" alt="twitter-logo"></a></li>
+        <li><a href=""><img src="image/medium2.png" alt="medium-logo"></a></li>
+    </ul>
```

Comment:
- [ ] Great job so far, kindly replicate the spacing provided on the design.
- [ ] Kindly change the background to `#ffffff` you can see the difference on the screenshot I provided below

### Your Website

![Screen Shot 2022-08-31 at 11 53 08 AM](https://user-images.githubusercontent.com/59416136/187639677-8eb155c3-af76-4d1d-9f89-93c7e48ba08a.png)

### Figma's Design
![Screen Shot 2022-08-31 at 11 59 39 AM](https://user-images.githubusercontent.com/59416136/187640879-70f727dd-33ee-4615-82d0-573a781eaedb.png)